src/packs/abilities/Shadow_ymzPghcR2SYV1TKr/Level_6_z97UrntTybKs668h/Black_Ashe_iHb7hZ3ns9oRJHEd/ability_Cinderstorm_oJV6wwbqG6OgBmqG.json

```
@@ -0,0 +1,68 @@
+{
+  "folder": "iHb7hZ3ns9oRJHEd",
+  "name": "Cinderstorm",
+  "type": "ability",
+  "_id": "oJV6wwbqG6OgBmqG",
+  "img": "icons/magic/fire/blast-jet-stream-embers-yellow.webp",
+  "system": {
+    "source": {
+      "book": "Heroes",
+      "page": "169",
+      "license": "Draw Steel Creator License"
+    },
+    "_dsid": "cinderstorm",
+    "story": "You teleport your friends in a burst of ash and fire.",
+    "keywords": [
+      "magic"
+    ],
+    "type": "maneuver",
+    "category": "heroic",
+    "resource": 9,
+    "trigger": "",
+    "distance": {
+      "type": "burst",
+      "primary": 4
+    },
+    "damageDisplay": "melee",
+    "target": {
+      "type": "selfAlly",
+      "custom": "",
+      "value": null
+    },
+    "power": {
+      "roll": {
+        "formula": "@chr",
+        "characteristics": [
+          "agility"
+        ]
+      },
+      "effects": {}
+    },
+    "effect": {
+      "before": "<p>Each target can teleport up to 5 squares. For each target in addition to you who teleports away from or into a space adjacent to an enemy, that enemy takes fire damage equal to your Agility score <span style=\"color:rgb(240, 246, 252);font-family:'Monaspace Neon', ui-monospace, SFMono-Regular, 'SF Mono', Menlo, Consolas, 'Liberation Mono', monospace;font-size:12px;font-style:normal;font-variant-ligatures:normal;font-variant-caps:normal;font-weight:400;letter-spacing:normal;orphans:2;text-align:start;text-indent:0px;text-transform:none;widows:2;word-spacing:0px;-webkit-text-stroke-width:0px;white-space:pre-wrap;background-color:rgba(46, 160, 67, 0.4);text-decoration-thickness:initial;text-decoration-style:initial;text-decoration-color:initial;display:inline !important;float:none\">[[/damage @A fire]]</span>. Additionally, a target who ends this movement in concealment or cover can use the Hide maneuver even if they are observed.</p>",
```

```suggestion
      "before": "<p>Each target can teleport up to 5 squares. For each target in addition to you who teleports away from or into a space adjacent to an enemy, that enemy takes fire damage equal to your Agility score [[/damage @A fire]]. Additionally, a target who ends this movement in concealment or cover can use the Hide maneuver even if they are observed.</p>",
```

Gotta watch that paste-with-formatting.

---

Sorry for that